crds/jwst/specs/nircam_filteroffset.rmap

```diff
@@ -0,0 +1,21 @@
+header = {
+    'derived_from' : 'miri_filteroffset 2020-04-28 16:00:00',
+    'file_ext' : '.asdf',
+    'filekind' : 'filteroffset',
+    'filetype' : 'FILTEROFFSET',
+    'instrument' : 'NIRCAM',
+    'mapping' : 'REFERENCE',
+    'name' : 'jwst_nircam_filteroffset_0000.rmap',
+    'observatory' : 'JWST',
+    'parkey' : (('META.INSTRUMENT.CHANNEL', 'META.INSTRUMENT.MODULE'), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
```

Comment:
What are the consequences of grouping these into pairs?

Reply:
That I actually don't know and higher authority will need to say. When not, there was a class api error. Most of the other jwst maps are this way also. i recall understanding awhile ago but don't remember now.

Reply:
The grouping of parameters follows the nesting of different Selector instances,  in this case (and almost always) the nesting is Match -> UseAfter and the parameters for UseAfter match the data,  not the reference file.  (So USEAFTER is compared to OBS-DATE and OBS-TIME,  modulo possibly misspelling those last two keywords.)   Did I understand the question correctly?